src/app/lib/analyticsUtils/eventListener/index.js

```diff
@@ -0,0 +1,49 @@
+import onClient from '../../utilities/onClient';
+import { getComponentsToTrack } from '..';
+import FrontPageAtiEventTracker from '../../../containers/ATIAnalytics/event/frontpage';
+
+const listener = (data, platform, statsDestination, service) => {
+  if (onClient()) {
+    // this is currently very limited to nav and brand so will need to be extended
+    // we could get values from the config for components that need tracking
+    // and pass them into `getComponentsToTrack which could then return
+    // an array like `trackedItems`
+    const navItems = getComponentsToTrack('track-nav');
+    const brandItems = getComponentsToTrack('track-brand');
```

Comment:
We could do this more generic as follows:
```
const trackedItems = ['track-nav', 'track-brand']; // get from conf

const componentsToTrack = Array.from(document.querySelectorAll(['data-track']))
    .filter(el => trackedItems.includes(el.dataset.track))
    .map(el => {
        el.addEventListener('click', event => {
           //...
        })
    })
)
```
Assuming that we use data attributes instead of classNames and that we set in the config which components are gonna be tracked. But, I have the same concern as Andrew said below.